app/assets/javascripts/application.js

```diff
@@ -16,7 +16,7 @@
 //= require ../../../node_modules/govuk_frontend_toolkit/javascripts/govuk/shim-links-with-button-role.js
 //= require ../../../node_modules/digitalmarketplace-frontend-toolkit/toolkit/javascripts/shim-links-with-button-role.js
 //= require ../../../node_modules/digitalmarketplace-frontend-toolkit/toolkit/javascripts/show-hide-content.js
-//= require ../../../node_modules/digitalmarketplace-frontend-toolkit/toolkit/javascripts/report-a-problem.js
+; // Hack needed because show-hide-content.js does not have a closing ";"
```

Comment:
I think we left it in 'just in case' we wanted to turn it back on again - I think it's been long enough now that we may as well remove it. It's in the history if we ever need it 😸